startup_runners/Makefile

```diff
@@ -19,14 +19,14 @@ startup_runner.class: startup_runner.java
 
 
 outer_startup_runner_c: outer_startup_runner.c
-	${CC} ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
-		${CPPFLAGS} outer_startup_runner.c \
+	${CC} --std=c99 ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
```

Comment:
If you build libkrun with c99, then weird things happen with time code:

```
cc   -Wall -shared -fPIC  \
         -Wall -pedantic -std=c99 libkruntime.c -o libkruntime.so \
          -lrt -ldl
libkruntime.c: In function ‘krun_clock_gettime_monotonic’:
libkruntime.c:144:29: error: storage size of ‘ts’ isn’t known
     struct timespec         ts;
                             ^
libkruntime.c:147:5: warning: implicit declaration of function ‘clock_gettime’ [-Wimplicit-function-declaration]
     if ((clock_gettime(ACTUAL_CLOCK_MONOTONIC, &ts)) < 0) {
     ^
libkruntime.c:27:35: error: ‘CLOCK_MONOTONIC_RAW’ undeclared (first use in this function)
 #define ACTUAL_CLOCK_MONOTONIC    CLOCK_MONOTONIC_RAW
                                   ^
libkruntime.c:147:24: note: in expansion of macro ‘ACTUAL_CLOCK_MONOTONIC’
     if ((clock_gettime(ACTUAL_CLOCK_MONOTONIC, &ts)) < 0) {
                        ^
libkruntime.c:27:35: note: each undeclared identifier is reported only once for each function it appears in
 #define ACTUAL_CLOCK_MONOTONIC    CLOCK_MONOTONIC_RAW
                                   ^
libkruntime.c:147:24: note: in expansion of macro ‘ACTUAL_CLOCK_MONOTONIC’
     if ((clock_gettime(ACTUAL_CLOCK_MONOTONIC, &ts)) < 0) {
                        ^
libkruntime.c:144:29: warning: unused variable ‘ts’ [-Wunused-variable]
     struct timespec         ts;
                             ^
libkruntime.c: In function ‘krun_get_fixed_pctr1_width’:
libkruntime.c:317:5: error: ‘asm’ undeclared (first use in this function)
     asm volatile(
     ^
libkruntime.c:317:9: error: expected ‘;’ before ‘volatile’
     asm volatile(
         ^
Makefile:23: recipe for target 'libkruntime.so' failed
```

I never got to the bottom of this, but instead just used gnu99.